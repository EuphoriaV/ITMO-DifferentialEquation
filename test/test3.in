2.57
(x - 8.48)
2.59
4.06