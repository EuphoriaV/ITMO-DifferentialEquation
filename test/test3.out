0.0024683